sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -358,8 +358,45 @@ The azure-core [reference documentation](https://azuresdkdocs.blob.core.windows.
 describes available configurations for retries, logging, transport protocols, and more.
 
 ## Troubleshooting
+
+### General
 The Text Analytics client will raise exceptions defined in [Azure Core](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-core/README.md).
 
+### Logging
+This library uses the standard
```

Comment:
Just curious, do other libraries follow the same logging pattern? is there chance this could be a general documentation where we can just point this section to?

Reply:
+1, Feel like at least in Java readme. We only have a brief description on logging: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/textanalytics/azure-ai-textanalytics#enable-client-logging

Reply:
The ref docs I've been pointed to show how to set up logging as an SDK developer, not a user. I think pointing to these will be more confusing than helpful. This is an open issue I need to bring up with the team.